requirements/requirements.md

```diff
@@ -1,20 +1,21 @@
 # Requirements
 
-A collection of requirements and scenarios, framing the scope of the Notary v2 project.
+A collection of requirements and scenarios, framing the scope of the Notary project.
 
 ## TOC
 
+<!-- no toc -->
 - [Goals](#goals)
 - [Non Goals](#non-goals)
 - [Scenarios](./scenarios.md)
-- [Threat Model](./threatmodel.md)
```

Comment:
It seems the threat model is gone. Is it intended?

Reply:
After I disable the TOC auto-generation, I missed this one. I will add it back.